.changeset/rotten-plants-peel.md

```diff
@@ -0,0 +1,5 @@
+---
+"@firebase/rules-unit-testing": patch
+---
+
+Fix typo in index.ts
```

Comment:
Could you say something more specific, like `getDoc` typo?

Reply:
Actually, the PR description does a pretty good job, but it will help us with the release if you also put it in the changeset:

```
This change fixes two typos in the documentation of the functions RulesTestEnvironment.authenticatedContext and RulesTestEnvironment.unauthenticatedContext of the @firebase/rules-unit-testing package.
```

Reply:
Thank you for the feedback. I just committed an updated an update changeset.

Reply:
Thank you so much!